frontend/public/components/catalog/catalog-item-icon.tsx

```diff
@@ -102,6 +103,7 @@ const logos = new Map()
   .set('icon-capedwarf', capedwarfImg)
   .set('icon-catalog', catalogImg)
   .set('icon-cassandra', cassandraImg)
+  .set('icon-che', cheImg)
```

Comment:
I'd prefer that we put the icon in our own package for now as it's not part of the catalog.